evap/staff/importers.py

```diff
@@ -243,11 +295,19 @@ def __init__(self):
         self.evaluations = {}
         self.enrollments = []
         self.names_de = set()
+        self.evaluation_data_factory = EvaluationDataFactory()
```

Comment:
nice³!

I feel like the class is quite long for what it does, but I don't see how it could be any shorter, so probably the problem is just a bit more complex than I'd intuitively assume...